src/Bridges/Variable/bridges/hermitian.jl

```diff
@@ -0,0 +1,350 @@
+so we can just ignore them.
+"""
+struct HermitianToSymmetricPSDBridge{T} <: AbstractBridge
+    variables::Vector{MOI.VariableIndex}
+    psd_constraint::MOI.ConstraintIndex{
+        MOI.VectorOfVariables,
+        MOI.PositiveSemidefiniteConeTriangle,
+    }
+    con_11_22::Vector{EQ{T}}
+    con12diag::Vector{EQ{T}}
+    con_12_21::Vector{EQ{T}}
+end
+
+const HermitianToSymmetricPSD{T,OT<:MOI.ModelLike} =
+    SingleBridgeOptimizer{HermitianToSymmetricPSDBridge{T},OT}
+
+function bridge_constrained_variable(
+    ::Type{HermitianToSymmetricPSDBridge{T}},
+    model::MOI.ModelLike,
+    set::MOI.HermitianPositiveSemidefiniteConeTriangle,
+) where {T}
+    n = set.side_dimension
+    variables, psd_constraint = MOI.add_constrained_variables(
+        model,
+        MOI.PositiveSemidefiniteConeTriangle(2n),
+    )
+
+    k11 = 0
+    k12 = MOI.dimension(MOI.PositiveSemidefiniteConeTriangle(n))
+    k21 = MOI.dimension(MOI.PositiveSemidefiniteConeTriangle(2n)) + 1
+    k22 = MOI.dimension(MOI.PositiveSemidefiniteConeTriangle(n))
+    X11() = variables[k11]
+    X12() = variables[k12]
+    function X21(i, j)
+        I = j
+        J = n + i
+        k21 = MOI.dimension(MOI.PositiveSemidefiniteConeTriangle(J - 1)) + I
+        return variables[k21]
+    end
+    X22() = variables[k22]
+    con_11_22 = EQ{T}[]
+    con12diag = EQ{T}[]
+    con_12_21 = EQ{T}[]
+    for j in 1:n
+        k22 += n
+        for i in 1:j
+            k11 += 1
+            k12 += 1
+            k22 += 1
+            push!(
+                con_11_22,
+                MOI.add_constraint(
+                    model,
+                    MOI.Utilities.operate(-, T, X11(), X22()),
+                    MOI.EqualTo(zero(T)),
+                ),
+            )
+            if i == j
+                push!(
+                    con12diag,
+                    MOI.add_constraint(
+                        model,
+                        convert(MOI.ScalarAffineFunction{T}, X12()),
+                        MOI.EqualTo(zero(T)),
+                    ),
+                )
+            else
+                push!(
+                    con_12_21,
+                    MOI.add_constraint(
+                        model,
+                        MOI.Utilities.operate(+, T, X21(i, j), X12()),
+                        MOI.EqualTo(zero(T)),
+                    ),
+                )
+            end
+        end
+        k12 += n
+    end
+
+    return HermitianToSymmetricPSDBridge(
+        variables,
+        psd_constraint,
+        con_11_22,
+        con12diag,
+        con_12_21,
+    )
+end
+
+const HermitianToSymmetricPSD{T,OT<:MOI.ModelLike} =
+    SingleBridgeOptimizer{HermitianToSymmetricPSDBridge{T},OT}
+
+function supports_constrained_variable(
+    ::Type{<:HermitianToSymmetricPSDBridge},
+    ::Type{MOI.HermitianPositiveSemidefiniteConeTriangle},
+)
+    return true
+end
+function MOI.Bridges.added_constrained_variable_types(
```

Comment:
```suggestion

function MOI.Bridges.added_constrained_variable_types(
```
Pet peeve: spaces between function definitions.